torch/_torch_docs.py

```diff
@@ -4272,6 +4272,59 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(x, y) -> Tensor
+
+Compute x*log(y) so that the result is 0 if x = 0.
+Similar to SciPy's `scipy.special.xlogy`.
+
+.. math::
+    \text{out}_i = x_i * \log{{y_i}}
+
+Args:
+    x (tensor): Multiplier
+    y (float or tensor): Argument
+
+.. note::
```

Comment:
See above discussion. If we keep this note move it so it's before the enumeration of arguments.

Reply:
I think this note can be removed now that the cases are enumerated.